pkg/disk_ops.go

```diff
@@ -396,12 +397,45 @@ func (r *diskOps) RemoveDMDevice(dmDevice string) error {
 	return err
 }
 
+// Make sure devices are ordered in a manner that ensures they can be deleted
+// When using thin provisioning we have to contend with _tdata and _tmeta dmsetup records
+// These cannot be deleted until the main dmsetup record is deleted
```

Comment:
Documentation comment does not follow golang conventions, should begin with the function name / present simple kind of phrasing